vertexai/evaluation/evaluate_output.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// evaluation package shows examples of working with Gen AI Evaluation API
+package evaluation
+
+// [START generativeaionvertexai_evaluation_pointwise]
```

Comment:
**TBD:** Adds a new region tag because the Python sample for this [doc section](https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/evaluation#python_1) [is located](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/generative_ai/model_tuning/create_evaluation_task_example.py#L22) under a different feature directory (`model_tuning`)

Reply:
Thank you, @telpirion ! I was hesitating how to proceed here, — hence the `TBD` tag 😸

Reply:
Updated!